test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -132,66 +133,130 @@ var _ = Describe("Mixedcpus", Ordered, func() {
+				Expect(cfg.Quota).To(Or(Equal("max"), Equal("-1")))
+			})
+			It("should have OPENSHIFT_ISOLATED_CPUS and OPENSHIFT_SHARED_CPUS env variables under the container", func() {
+				rl := &corev1.ResourceList{
+					corev1.ResourceCPU:    resource.MustParse("1"),
+					corev1.ResourceMemory: resource.MustParse("100Mi"),
+					sharedCpusResource:    resource.MustParse("1"),
+				}
+				p, err := createPod(ctx, testclient.Client, testutils.NamespaceTesting,
+					withRequests(rl),
+					withLimits(rl),
+					withRuntime(components.GetComponentName(profile.Name, components.ComponentNamePrefix)))
+				Expect(err).ToNot(HaveOccurred())
+				cmd := printMixedCPUsEnvCmd()
+				output, err := pods.ExecCommandOnPod(testclient.K8sClient, p, "", cmd)
+				Expect(err).ToNot(HaveOccurred(), "failed to execute command on pod; cmd=%q pod=%q", cmd, client.ObjectKeyFromObject(p).String())
+				isolatedAndShared := strings.Split(string(output), "\r\n")
+				isolated, err := cpuset.Parse(isolatedAndShared[0])
+				Expect(err).ToNot(HaveOccurred())
+				Expect(isolated.IsEmpty()).ToNot(BeTrue())
+				shared, err := cpuset.Parse(isolatedAndShared[1])
+				ppShared, _ := cpuset.Parse(string(*profile.Spec.CPU.Shared))
+				Expect(err).ToNot(HaveOccurred())
+				Expect(shared.Equals(ppShared)).To(BeTrue(), "OPENSHIFT_SHARED_CPUS value not equal to what configure in the performance profile."+
+					"OPENSHIFT_SHARED_CPUS=%s spec.cpu.shared=%s", shared.String(), ppShared.String())
+			})
 		})
-		It("should have OPENSHIFT_ISOLATED_CPUS and OPENSHIFT_SHARED_CPUS env variables under the container", func() {
-			rl := &corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("1"),
-				corev1.ResourceMemory: resource.MustParse("100Mi"),
-				sharedCpusResource:    resource.MustParse("1"),
-			}
-			p, err := createPod(ctx, testclient.Client, testutils.NamespaceTesting,
-				withRequests(rl),
-				withLimits(rl),
-				withRuntime(components.GetComponentName(profile.Name, components.ComponentNamePrefix)))
-			Expect(err).ToNot(HaveOccurred())
-			cmd := printMixedCPUsEnvCmd()
-			output, err := pods.ExecCommandOnPod(testclient.K8sClient, p, "", cmd)
-			Expect(err).ToNot(HaveOccurred(), "failed to execute command on pod; cmd=%q pod=%q", cmd, client.ObjectKeyFromObject(p).String())
-			isolatedAndShared := strings.Split(string(output), "\r\n")
-			isolated, err := cpuset.Parse(isolatedAndShared[0])
-			Expect(err).ToNot(HaveOccurred())
-			Expect(isolated.IsEmpty()).ToNot(BeTrue())
-			shared, err := cpuset.Parse(isolatedAndShared[1])
-			ppShared, _ := cpuset.Parse(string(*profile.Spec.CPU.Shared))
-			Expect(err).ToNot(HaveOccurred())
-			Expect(shared.Equals(ppShared)).To(BeTrue(), "OPENSHIFT_SHARED_CPUS value not equal to what configure in the performance profile."+
-				"OPENSHIFT_SHARED_CPUS=%s spec.cpu.shared=%s", shared.String(), ppShared.String())
+
+		When("Modifying the shared CPUs in the performance profile", func() {
```

Comment:
Can we add tags/labels to differentiate between spec blocks that check mixed-CPUs configuration and those that apply updates (slow and require reboot)?

Reply:
Sure. Is there a list of standardize labels that we should use?

Reply:
We can continue the discussion here: https://github.com/openshift/cluster-node-tuning-operator/pull/936